ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -126,14 +139,11 @@ export const processWaitingQueue = async (department) => {
 
 	if (room && room.servedBy) {
 		const { _id: rid, servedBy: { _id: agentId } } = room;
-		helperLogger.debug(`Inquiry ${ inquiry._id } taken succesfully by agent ${ agentId }. Notifying`);
+		helperLogger.debug(`Inquiry ${ inquiry._id } taken successfully by agent ${ agentId }. Notifying`);
 		return setTimeout(() => {
 			propagateAgentDelegated(rid, agentId);
 		}, 1000);
 	}
-
-	const { departmentId } = room || {};
-	await debouncedDispatchWaitingQueueStatus(departmentId);
```

Comment:
FYI: This method was being called whenever the waiting queue was processed but there's no reason for that.